slack-base/src/main/java/com/hubspot/slack/client/methods/params/conversations/ConversationsJoinParamsIF.java

```diff
@@ -0,0 +1,18 @@
+package com.hubspot.slack.client.methods.params.conversations;
+
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ConversationsJoinParamsIF {
+
+  /*
+   * ID of channel to join
+   */
+  String getChannel();
```

Comment:
Let's name this `getChannelId` to make clear it's an ID (without needing the comment), and use `@JsonProperty` to rename it to what slack expects. In general we prefer to avoid needing comments, since they tend to indicate unclear/inadequate naming